src/layout.hoc.js

```diff
@@ -0,0 +1,132 @@
+// @flow
```

Comment:
I figured it'd be nice to have. But I don't really have a use case for it. So I'm fine dumping it